pkg/sdk/testint/alerts_integration_test.go

```diff
@@ -1,6 +1,7 @@
 package testint
 
 import (
+	"errors"
```

Comment:
Addin a general comment here, to make it easier to respond to.

There are two things I noticed during checking the completeness:
- I checked that there is one more object: session policies. The docs https://docs.snowflake.com/en/sql-reference/sql/show-session-policies does not contain any filtering: is this really not possible or is it the error in the docs?
- application roles - there is no LIKE used in the ShowByID method. I know this is different topic but because we are touching almost all ShowByID, maybe we can also improve this one. WDYT?

Reply:
SHOW SESSION POLICIES LIKE 'session_policy_prod_1'
This does work. So it is a documentation issue. Wrote in docs-discuss channel here: https://snowflake.slack.com/archives/C6380540P/p1712101727265659

Application roles... i suppose I can come back to this one.

Reply:
Thanks!